### ISSUE TITLE
cheerio

### ISSUE THREAD

**Issue by naushadkm** [hemanth/nmotw.in#15]
Tiny, fast, and elegant implementation of core jQuery designed specifically for the server.

var cheerio = require('cheerio'),
    $ = cheerio.load('\<h2 class="title">Hello world\</h2>');
 
$('h2.title').text('Hello there!');
$('h2').addClass('welcome');
 
$.html(); //=> \<h2 class="title welcome">Hello there!\</h2>

**Comment by hemanth** [comment]
Yup, `cheerio` was supposed to be the first module :+1: just digging into those modules that are not in the lime light, but yet very useful :)
